src/incremental/compareAST.ml

```diff
@@ -157,7 +182,10 @@ and eq_compinfo (a: compinfo) (b: compinfo) (acc: (typ * typ) list) =
   a.cdefined = b.cdefined (* Ignore ckey, and ignore creferenced *)
 
 and eq_fieldinfo (a: fieldinfo) (b: fieldinfo) (acc: (typ * typ) list)=
-  a.fname = b.fname && eq_typ_acc a.ftype b.ftype acc && a.fbitfield = b.fbitfield &&  eq_list eq_attribute a.fattr b.fattr
+  if Messages.tracing then Messages.tracei "compare" "fieldinfo %s vs %s\n" a.fname b.fname;
+  let r = a.fname = b.fname && eq_typ_acc a.ftype b.ftype acc && a.fbitfield = b.fbitfield &&  eq_list eq_attribute a.fattr b.fattr in
+  if Messages.tracing then Messages.traceu "compare" "fieldinfo %s vs %s\n" a.fname b.fname;
+  r
```

Comment:
Should we keep this debugging output?

Reply:
I think it's worth keeping because `eq_type_acc` and `eq_fieldinfo` are essentially the two things which can be used to get an understanding of how this works and what it skips, in case something about it ever needs to be fixed again.

I'll just rename the tracing key from `compare` to `compareast` to be a bit more specific.